login.php

```diff
@@ -46,8 +49,23 @@
     header("Location: {$URL}");
     exit;
 }
-//VALIDATE LOGIN INFORMATION
 else {
+    // Custom login loader
+    if (!$session->exists('login_custom.php')) {
+        $globals = [
+            'guid'        => $guid,
+            'connection2' => $connection2,
+            'username' => $username,
+            'password' => $password
+        ];
+
+        $session->set('login_custom.php', $page->fetchFromFile('./login_custom.php', $globals));
```

Comment:
I wonder if `fetchFromFile` may not work as you're expecting for this situation. It's primary use is for templating, as it runs some code and returns the string output. This line of code would run the PHP in login_custom.php and only cache the output result. Then line 66 would echo the cached output without running the PHP code.

Since your aim is likely to run the PHP code for every login execution, a `require_once` may be better in this instance.

Reply:
That would make a lot more sense, you're right. Pushing the changes now!